Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -375,7 +375,7 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver,
     }
 
     func presentAudioSourcePicker() {
-        AssertIsOnMainThread()
```

Comment:
We were often confusing `AssertIsOnMainThread()` which is intended for Swift code and `OWS AssertIsOnMainThread()` which is intended for Obj-C, so I've renamed the former to make clear that it is for swift usage only.

Reply:
Well, the asserts compile out entirely in release builds thanks to the preprocessor.  And we use these asserts all over the place, so it's nice to eliminate them.